insights/parsers/insights_client_tags.py

```diff
@@ -0,0 +1,52 @@
+"""
+InsightsClientTags - file ``/etc/insights-client/tags.yaml``
+=======================================================================
+"""
+
+from insights import parser
+from insights.core import YAMLParser
+from insights.specs import Specs
+
+
+@parser(Specs.insights_client_tags)
+class InsightsClientTags(YAMLParser):
+    """
+    This class parses the YAML-formatted file `/etc/insights-client/tags.yaml`.
+    The file is created by the user after running this command:
+
+    `insights-client --group=<group-name>`
+
+    The command above creates the file and adds an initial group tag
+    as defined by the `--group` option.
+    Other tags are added manually by the user.
+
+    Sample input data:
```

Comment:
You may want to use a double colon `::` to put the following data into a literal block 

see: https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html#example-google